.eslintrc.yml

```diff
@@ -29,3 +29,11 @@ rules:
   # https://github.com/benmosher/eslint-plugin-import/issues/1615#issuecomment-577500405
   import/extensions:
     ['error', 'ignorePackages', { js: 'never', ts: 'never', jsx: 'never' }]
+  prefer-template: "off"
+  class-methods-use-this: "off"
+  semi: [2, "always"]
+  no-cond-assign: [2, "except-parens"]
+  no-restricted-syntax: [0]
+  no-await-in-loop: [0]
+  no-bitwise: [0]
+  no-trailing-spaces: [2]
```

Comment:
I these added to the .eslintrc file because your existing TypeScript code doesn't follow these rules, and I didn't want to follow them in my new files.

Separately, it doesn't look like you're including any of your TypeScript files when you lint. I'd be willing to clean things up and start using ESLint on your TypeScript files if you're open to it.